packages/react-day-picker/src/DayPicker.test.tsx

```diff
@@ -1,3 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { freezeBeforeAll } from '@test/utils';
+import { getDayButton } from '@test/po';
+import { DayPicker } from './DayPicker';
+
+const today = new Date(2022, 5, 10);
+const tomorrow = new Date(2022, 5, 11);
+freezeBeforeAll(today);
+
+describe('when rendering a month that contains today', () => {
+  beforeEach(() => render(<DayPicker month={today} mode="single" />));
+  test('it should add the default class name for today', () => {
+    expect(getDayButton(today)).toHaveClass('rdp-day_today');
```

Comment:
I think this is a nice high-level test that tests not the boolean value for the `today` modifier status, but that the class name is applied...seems ok to keep from my perspective, even though we also have a test for modifiers-today.tsx